python/cuda_parallel/CMakeLists.txt

```diff
@@ -71,7 +65,7 @@ set(_generated_extension_src "${cuda_parallel_BINARY_DIR}/_bindings.c")
 set(_depfile "${cuda_parallel_BINARY_DIR}/_bindings.c.dep")
 add_custom_command(
     OUTPUT "${_generated_extension_src}"
-    COMMAND ${CYTHON_EXECUTABLE}
+    COMMAND "${Python3_EXECUTABLE}" -m cython
```

Comment:
Just a note that if we ever have a need to use Python debug builds on Windows that we'd need to use `${Python3_INTERPRETER}` here instead (it uses a generator expression to handle switching automatically).

Reply:
Another option would be something like:

```
if (CMAKE_VERSION VERSION LESS 3.30)
  // use _EXECUTABLE
else()
  // use _INTERPRETER
endif()
```